Tools/MonoGame.Content.Builder/MonoGame.Content.Builder.csproj

```
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net452</TargetFrameworks>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Content Include="MonoGame.Content.Builder.props" PackagePath="build" />
+    <Content Include="MonoGame.Content.Builder.targets" PackagePath="build" />
+    <Content Include="version\Version.props" PackagePath="build\version\$(Version)" />
```

Nuget uses PackageVersion for the version. If not specified it will take on the value of the Version property, but it can be set directly, so let's use PackageVersion instead of Version here.

```suggestion
    <Content Include="version\Version.props" PackagePath="build\version\$(PackageVersion)" />
```